test/integration.test.js

```diff
@@ -50,17 +50,18 @@ test.serial('Verify GitLab auth', async t => {
 test.serial('Publish a release', async t => {
   const owner = 'test_user';
   const repo = 'test_repo';
+  const repoId = `${owner}%2F${repo}`;
```

Comment:
No need to create this extra variable as `repoId` should be used only in the URI, not in the payload.